configs/partials/webpack.config.base.js

```diff
@@ -11,12 +11,16 @@ const { dependencies: externals } = require('../../app/package.json');
 
 const extensions = ['.js', '.jsx', '.json', '.ts', '.tsx'];
 
-if (
-    process.env.TARGET_NET &&
-    !['stagenet', 'testnet'].includes(process.env.TARGET_NET)
-) {
+const targetNet = process.env.TARGET_NET;
+
+let userData = process.env.npm_package_userData;
```

Comment:
I don't see the benefit of having `userData` defined in `package.json`. I think we might as well just set it here.

Reply:
I've moved it.
I was probably thinking it would be easier to find there, but it should never have to be changed anyway. :shrug:

Reply:
Well... If we could migrate it easily, then we should do that now. And otherwise it is quite likely remaining quite static.